src/emulator/apphosting/config.ts

```diff
@@ -15,27 +16,33 @@ export async function getLocalAppHostingConfiguration(
 ): Promise<AppHostingYamlConfig> {
```

Comment:
We should update the JSDoc to reflect the new behaviour of us loading in the apphosting.emulator.yaml file as well.